catalog/ui/catalog-ui-search/src/main/java/org/codice/ddf/catalog/ui/config/ConfigurationApplication.java

```diff
@@ -860,4 +866,20 @@ public String getMapHome() {
   public void setMapHome(String mapHome) {
     this.mapHome = mapHome;
   }
+
+  public boolean isDisableSearchHistorical() {
```

Comment:
✏️  Refactoring to `isSearchHistoricalDisabled` would make this method name read better, but it's probably not worth it considering the other refactoring that would be needed to keep the terminology consistent

Reply:
I agree with @mackncheesiest that it'd be nice if we could improve these method names. How bad would it be to use `isHistoricalSearchDisabled()` and `setHistoricalSearchDisabled()`, `isArchiveSearchDisabled()` and `setArchiveSearchDisabled()` instead?

Reply:
Also, to keep things consistent between the UI and the backend, and to avoid double negations (i.e., `!is...Disabled()`), should we use `Enabled` instead of `Disabled` here as well and default the values to `true` in the metatype and config files?